hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/offset/OffsetQueue.java

```diff
@@ -1,25 +1,23 @@
 package pl.allegro.tech.hermes.consumers.consumer.offset;
 
 import org.jctools.queues.MessagePassingQueue;
+import pl.allegro.tech.hermes.api.Subscription;
 import pl.allegro.tech.hermes.common.config.ConfigFactory;
 import pl.allegro.tech.hermes.common.config.Configs;
 import pl.allegro.tech.hermes.common.metric.HermesMetrics;
 import pl.allegro.tech.hermes.consumers.queue.MonitoredMpscQueue;
 
-import javax.inject.Inject;
-
 public class OffsetQueue {
 
     private final MonitoredMpscQueue<SubscriptionPartitionOffset> inflightOffsetsQueue;
 
     private final MonitoredMpscQueue<SubscriptionPartitionOffset> commitOffsetsQueue;
 
-    @Inject
     public OffsetQueue(HermesMetrics metrics, ConfigFactory configFactory) {
```

Comment:
This relates to my other comment in the actual gauge creation in the queue. In the end under the umbrella term `committedOffsets` and `inflightOffsets` just one of the queues would be metered.